pkg/errs/errno.go

```diff
@@ -93,3 +93,8 @@ var (
 	ErrLoadValue  = errors.Normalize("load value from etcd failed", errors.RFCCodeText("PD:etcdutil:ErrLoadValue"))
 	ErrGetCluster = errors.Normalize("get cluster from remote peer failed", errors.RFCCodeText("PD:etcdutil:ErrGetCluster"))
 )
+
+// etcd errors
```

Comment:
Do we need to classify this error as a server error?

Reply:
@rleungx What do you think?

Reply:
I think we may talk with the issue in new pr, we should merge it as soon as possible to improve ci

Reply:
For the error which is generated from the third-party project, we can use the project name instead of our package name.